webstart.ts

```diff
@@ -41,7 +76,42 @@ function webStart() {
     ).then(bytes => 
       WebAssembly.instantiate(bytes, { js: { mem: memory } })
     );
+    
+    function console_log_class(repl:BasicREPL, pointer:number, classname:string,level:number) : Array<string>{
 
+      var fields_offset_ = repl.currentEnv.classes.get(classname);
+      var fields_type = repl.currentTypeEnv.classes.get(classname)[0];
+      var mem = new Uint32Array(memory.buffer);
```

Comment:
I wonder if there's a way to call `load` and `store` rather than doing direct indexing here. This will be hard to maintain if memory management updates representations.

Reply:
sorry just found this, I understand the load part, what do you mean by the store part?

Reply:
load to get the object value out and what about store?